test/unit/uk/gov/hmrc/apinotificationqueue/controller/QueueControllerSpec.scala

```diff
@@ -18,22 +18,33 @@ package uk.gov.hmrc.apinotificationqueue.controller
 
 import play.api.http.Status
 import play.api.test.FakeRequest
-import play.api.http.Status
-import play.api.test.FakeRequest
-import uk.gov.hmrc.play.test.UnitSpec
-import uk.gov.hmrc.play.test.WithFakeApplication
 import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
 
-class MicroserviceHelloWorldControllerSpec extends UnitSpec with WithFakeApplication{
+class QueueControllerSpec extends UnitSpec with WithFakeApplication {
 
-  val fakeRequest = FakeRequest("GET", "/")
 
-  "GET /" should {
+  "GET /messages" should {
     "return 200" in {
-      val controller = new MicroserviceHelloWorld()
-      val result = controller.hello()(fakeRequest)
+      val controller = new Queue()
+      val result = controller.getAll()(FakeRequest("GET", "/messages"))
       status(result) shouldBe Status.OK
     }
   }
 
+  "GET /message/:id" should {
```

Comment:
Shall we add also a 404 case?

Reply:
It's currently hard coded to return so when we actually do the logic yes.